src/index.html

```diff
@@ -1,14 +1,20 @@
-<!doctype html>
+<!DOCTYPE html>
 <html lang="en">
   <head>
-    <meta charset="UTF-8">
-    <meta name="viewport"
-          content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0">
-    <meta http-equiv="X-UA-Compatible" content="ie=edge">
+    <meta charset="UTF-8" />
+    <meta
+      name="viewport"
+      content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0"
+    />
+    <meta http-equiv="X-UA-Compatible" content="ie=edge" />
     <title>Stop watch</title>
-    <link rel="stylesheet" href="./styles/main.scss">
+    <link rel="stylesheet" href="./styles/main.scss" />
   </head>
-  <body>
-    <h1>Stop watch</h1>
+  <body class="page">
+    <section class="page__watch watch">
+      <div class="watch__center"></div>
```

Comment:
```suggestion
      <div class="watch__center" />
```
but here you may use self-close elements

Reply:
Also.. Linter doesn't miss.. But, thanks!)